src/Examples/Advanced/CustomProperties/Parameter_estimation_PR.ipynb

```diff
@@ -191,14 +191,13 @@
      "name": "stdout",
      "output_type": "stream",
      "text": [
-      "2021-05-14 15:33:37 [INFO] idaes.generic_models.properties.core.generic.generic_property: DEPRECATED - definiton of generic property packages is moving to using static classes instead of modules. Please refer to the IDAES documentation.\n",
-      "2021-05-14 15:33:37 [INFO] idaes.init.fs.state_block: Starting initialization\n",
-      "2021-05-14 15:33:37 [INFO] idaes.init.fs.state_block: Dew and bubble point initialization: optimal - Optimal Solution Found.\n",
-      "2021-05-14 15:33:37 [INFO] idaes.init.fs.state_block: Equilibrium temperature initialization completed.\n",
-      "2021-05-14 15:33:37 [INFO] idaes.init.fs.state_block: State variable initialization completed.\n",
-      "2021-05-14 15:33:38 [INFO] idaes.init.fs.state_block: Phase equilibrium initialization: infeasible - Converged to a locally infeasible point. Problem may be infeasible..\n",
-      "2021-05-14 15:33:38 [INFO] idaes.init.fs.state_block: Property initialization: infeasible - Converged to a locally infeasible point. Problem may be infeasible..\n",
-      "2021-05-14 15:33:38 [INFO] idaes.init.fs.state_block: Property package initialization: infeasible - Converged to a locally infeasible point. Problem may be infeasible..\n"
+      "2021-07-28 17:30:43 [INFO] idaes.init.fs.state_block: Starting initialization\n",
+      "2021-07-28 17:30:44 [INFO] idaes.init.fs.state_block: Dew and bubble point initialization: optimal - Optimal Solution Found.\n",
+      "2021-07-28 17:30:44 [INFO] idaes.init.fs.state_block: Equilibrium temperature initialization completed.\n",
+      "2021-07-28 17:30:44 [INFO] idaes.init.fs.state_block: State variable initialization completed.\n",
+      "2021-07-28 17:30:44 [INFO] idaes.init.fs.state_block: Phase equilibrium initialization: infeasible - Converged to a locally infeasible point. Problem may be infeasible..\n",
+      "2021-07-28 17:30:44 [INFO] idaes.init.fs.state_block: Property initialization: infeasible - Converged to a locally infeasible point. Problem may be infeasible..\n",
+      "2021-07-28 17:30:44 [INFO] idaes.init.fs.state_block: Property package initialization: infeasible - Converged to a locally infeasible point. Problem may be infeasible..\n"
```

Comment:
Just to check - I see an infeasible solution here. Is this a problem for the notebook/example?

If it isn't an issue, do we have a note to tell the reader not to worry and why?

Reply:
I changed the initial conditions to avoid any confusion or the need to add a note.